arviz/plots/hdiplot.py

```diff
@@ -87,14 +86,20 @@ def plot_hdi(
 
     Examples
     --------
-    Plot HDI interval of simulated regression data using `y` argument:
+    Plot HDI interval of simulated random-walk data using `y` argument:
 
     .. plot::
         :context: close-figs
 
         >>> import numpy as np
         >>> import arviz as az
-        >>> x_data = np.random.normal(0, 1, 100)
+        >>> # time-steps random walk
+        >>> x_data = range(0, 100)
+        >>> # Mean random walk
+        >>> mu = np.zeros(100)
+        >>> for i in x_data:
+        >>>    mu[i] = mu[i-1] + np.random.normal(0, 1, 1)
+        >>> # Simulated pp samples form the random walk time series
         >>> y_data = np.random.normal(2 + x_data * 0.5, 0.5, size=(2, 50, 100))
         >>> az.plot_hdi(x_data, y_data)
```

Comment:
I don't understand what is supposed to happen here, and I am therefore not sure on how to fix the code. It currently isn't generating any plot: https://arviz--2211.org.readthedocs.build/en/2211/api/generated/arviz.plot_hdi.html.

Reply:
I just update the example